packages/core/src/config/buildSchemaWithPluginContributions.ts

```diff
@@ -0,0 +1,27 @@
+import { commonTokens, PluginResolver, tokens } from '@stryker-mutator/api/plugin';
+import { Logger } from '@stryker-mutator/api/logging';
+
+import { coreTokens } from '../di';
+
+function mergedSchema(mainSchema: any, additionalSchemas: any[]): object {
```

Comment:
aren't these `object`s? (a bit more strict than any)

Reply:
Yeah, but I didn't want to bother with trying to explain TypeScript that it has `properties` and `definitions`. I know they have them and it is a private API for the time being. Didn't want to waste time on writing code to validate that the schemas are valid json schemas.